vllm/executor/gpu_executor.py

```diff
@@ -168,6 +168,12 @@ def check_health(self) -> None:
         # it's running.
         return
 
+    def start_profile(self) -> None:
```

Comment:
Maybe, we should also add start_profile/stop_profile for CPU-only targets?

Reply:
+1, either that or make it clear in the documentation & example that this is currently only supported on GPUs

Reply:
Thanks for the suggestion, missed it. @DamonFool @ywang96 added to `cpu_executor.py` in [2b23e8f](https://github.com/vllm-project/vllm/pull/7943/commits/2b23e8f3dfc4a359ab62e47c009b393dcc7a8f56) PTAL.